cmd/livepeer/livepeer.go

```diff
@@ -183,20 +191,23 @@ func main() {
 	} else if transcoder {
 		n.NodeType = core.TranscoderNode
 		glog.Info("***Livepeer is in transcoder mode ***")
-		server.RunTranscoder(n, *orchAddr)
+		if len(orchAddresses) > 0 {
+			server.RunTranscoder(n, orchAddresses[0])
+		}
 		return
 	} else {
 		n.NodeType = core.BroadcasterNode
 	}
 
 	if n.NodeType == core.BroadcasterNode {
-		if *orchAddr == "" {
-			glog.Info("No orchestrator specified; transcoding will not happen")
+		if len(orchAddresses) <= 0 {
+			n.OrchestratorSelector = discovery.NewOffchainOrchestratorFromOnchainList(n)
 		} else {
-			n.OrchestratorSelector = discovery.NewOffchainOrchestrator(n, *orchAddr)
-			if n.OrchestratorSelector == nil {
-				return
-			}
+			n.OrchestratorSelector = discovery.NewOffchainOrchestrator(n, orchAddresses)
+		}
+		if n.OrchestratorSelector == nil {
+			glog.Errorf("No orchestrator specified; transcoding will not happen")
+			return
```

Comment:
This will quit early for offchain mode; we can just continue running instead of returning.